src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -243,4 +241,13 @@ private void CheckEntityNamePresent(string entityName)
                 subStatusCode: DataApiBuilderException.SubStatusCodes.EntityNotFound);
         }
     }
+
+    private void SetupDataSourcesUsed()
+    {
+        SqlDataSourceUsed = _dataSourceNameToDataSource.Values.Any
+            (x => x.DatabaseType == DatabaseType.MSSQL || x.DatabaseType == DatabaseType.PostgreSQL || x.DatabaseType == DatabaseType.MySQL);
```

Comment:
```suggestion
            (x => x.DatabaseType is DatabaseType.MSSQL || x.DatabaseType is DatabaseType.PostgreSQL || x.DatabaseType is DatabaseType.MySQL);
```
nit: we favor `is` over `==` when possible

Reply:
fixed.